### ISSUE TITLE
[Feature Request] Allow changing the tag for VBottomNav

### ISSUE THREAD

**Issue by rudolfbyker** [vuetifyjs/vuetify#11392]
### Problem to solve
Screen readers like having the main site navigation in a `<nav>` element. See https://dequeuniversity.com/rules/axe/3.5/region . This is also good semantic HTML5 practice.

### Proposed solution
Some Vuetify components allow changing their root tag from `div` to something else through the `tag` prop. VBottomNav should also support that prop.

**Comment by rudolfbyker** [comment]
I'm not familiar with the Vuetify source code or Typescript, but I did some digging and it looks like the `div` is hard-coded in `VItemGroup.ts` on line `251`, so this would have to start with making that configurable, if VBottomNavigation is to continue extending that.

**Comment by johnleider** [comment]
Correct, this should be updated in VItemGroup. Quick win for v2.4.x

**Comment by rudolfbyker** [comment]
So has been updated in `VItemGroup`, but it's not in `VBottomNav` yet, so I think the issue should be re-opened.

**Comment by johnleider** [comment]
Good catch @rudolfbyker . Pushed a fix in 4d627e9d0
